modules/flowable-task-service/src/main/resources/org/flowable/task/service/db/mapping/entity/Task.xml

```diff
@@ -556,6 +556,13 @@
       <if test="processInstanceId != null">
         and RES.PROC_INST_ID_ = #{processInstanceId}
       </if>
+      <if test="taskIdList != null &amp;&amp; taskIdList.size() &gt; 0">
+        and RES.ID_ IN
+        <foreach item="taskId" index="index" collection="taskIdList"
```

Comment:
We recently did a change when checking for a size of a list. please use `!taskIdList.empty` instead.